doc/about.rst

```diff
@@ -3,7 +3,7 @@
 About Alex
 ==========
 
-Alex can always be obtained from its `home page <http://www.haskell.org/alex>`__.
```

Comment:
I think this just needs to be changed to `https`, since there are redirects: https://github.com/haskell-infra/www.haskell.org#subsites